LLMonFHIR/FHIRInterpretation/UserStudy/UserStudyCodes.swift

```diff
@@ -0,0 +1,45 @@
+//
+// This source file is part of the Stanford Spezi project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Spezi
+import SpeziAccessGuard
+import SpeziLocalStorage
+
+
+@MainActor
+final class UserStudyCodes: Module, Sendable {
+    @Dependency(LocalStorage.self) private var localStorage
```

Comment:
Could we use a secure storage setting here? Would also enable us to persist things across different app launches 👍